deploy/vagrant/provision/roles/tachyon/tasks/config.yml

```diff
@@ -18,6 +18,14 @@
   script: zookeeper.sh
   when: tachyon_masters > 1
 
+- name: set journal folder as a shared folder in underfs for fault tolerance
+  script: set_journal_folder.sh
+  when: tachyon_masters > 1
```

Comment:
I think even in the single master setup, the journal should write to UFS. Or maybe, there should be a config for it in ufs.yml?